.github/workflow_scripts/build_image_prediction_tutorial.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -ex
+
+BRANCH=$(basename $1)
+GIT_REPO=$2
+COMMIT_SHA=$3
+PR_NUMBER=$4  # For push events, PR_NUMBER will be empty
+
+source $(dirname "$0")/env_setup.sh
+source $(dirname "$0")/write_to_s3.sh
+
+setup_build_contrib_env
+setup_mxnet_gpu
```

Comment:
Do we still use mxnet for image prediction?

Reply:
It is in our Jenkins currently. Not 100% if our docs use it at all

Reply:
We are still using `ImageDataset` from GluonCV to load dataset for both image predictor tutorial and newest AutoMM tutorial. We are going to refactor this after the integration of MMDetection.